lib/Raven/Client.php

```diff
@@ -1359,6 +1362,11 @@ public function onShutdown()
         if (!defined('RAVEN_CLIENT_END_REACHED')) {
             define('RAVEN_CLIENT_END_REACHED', true);
         }
+        if ($this->fastcgi_finish_request
+          && !empty($this->_pending_events)
+          && function_exists('fastcgi_finish_request')) {
+          fastcgi_finish_request();
```

Comment:
FastCGI/backend control should not be placed in logging library. I'm afraid you should add your handler for register_shutdown_function by yourself. As I do https://github.com/nokitakaze/ascetcms/blob/concept/functions/ascetcmsengine.php#L518